bin/install-dependencies.sh

```diff
@@ -19,6 +19,10 @@ pipopt='--exists-action=b'
 
 $pip install $pipopt -U pip
 
+$pip install $pipopt -U $(curl -s https://raw.githubusercontent.com/ckan/ckan/$ckan_sha/requirement-setuptools.txt)
+$pip install $pipopt -r https://raw.githubusercontent.com/ckan/ckan/$ckan_sha/requirements.txt
```

Comment:
Is there a reason why this one is `-r` instead of `-U $(curl` like the others? I can't see any other examples like this in the file.

Reply:
I'm not sure, I just moved the code up as there was a dependency clash to do with `enum34` so I haven't actually changed how it's installed